database/services/db.go

```diff
@@ -16,7 +16,11 @@ func open() (db *leveldb.DB, err error) {
 	instanceMutex.Lock()
 	defer instanceMutex.Unlock()
 	if _instance == nil {
-		storagePath := filepath.Join(config.Path, "database", "services")
+		c, err := config.Global()
+		if err != nil {
+			return nil, err
+		}
+		storagePath := filepath.Join(c.Core.Path, "database", "services")
```

Comment:
maybe keep this path in config package as well?

Reply:
not related to this PR, we can do that in a separate issue/pr maybe the #481